pulp_deb/app/settings.py

```diff
@@ -7,3 +7,5 @@
 
 FORBIDDEN_CHECKSUM_WARNINGS = True
 FORCE_IGNORE_MISSING_PACKAGE_INDICES = False
+
+APT_BY_HASH = True
```

Comment:
Nitpick: For lack of a better organizing principle, let's put these in alphabetical order:
```suggestion
APT_BY_HASH = True
FORBIDDEN_CHECKSUM_WARNINGS = True
FORCE_IGNORE_MISSING_PACKAGE_INDICES = False
```

Reply:
Sure thing.